Resources/Plists/Info.plist

```diff
@@ -382,6 +384,35 @@
 				</array>
 			</dict>
 		</dict>
+		<dict>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>com.apple.package</string>
+			</array>
+			<key>UTTypeDescription</key>
+			<string>Sequel Ace Bundle</string>
+			<key>UTTypeIconFile</key>
+			<string></string>
+			<key>UTTypeIcons</key>
+			<dict>
+				<key>UTTypeIconBackgroundName</key>
+				<string></string>
+				<key>UTTypeIconBadgeName</key>
+				<string>Document_icon</string>
+				<key>UTTypeIconText</key>
+				<string>SPBUNDLE</string>
```

Comment:
This should probably be SABUNDLE (or just BUNDLE for .spbundle and .sabundle both) @jamesstout @Kaspik

Reply:
no idea!